brave/brave-core
# Disables Ads UI in 0.59 build

Fixes: https://github.com/brave/brave-browser/issues/3077
Partially picks in: https://github.com/brave/brave-core/pull/1283

Changes in #1283 were picked in as we are getting the latest `DisabledContent` component which has its own icons backed in, so unused ones in this branch are now removed. We want to be using the newest Ads megaphone icon.

This disables the Ads UI from being interacted with, there is also a temporary patch to disable Ads for anyone who had it previously enabled on this version

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
1. Build 0.59.x
2. Enable Rewards, and Ads
3. Using the same profile, build from this PR
4. Ensure that the Ads settings UI is disabled
5. Ensure that Ads service is disabled

## Reviewer Checklist:
